tool/tsh/common/proxy.go

```diff
@@ -341,6 +343,59 @@ func maybeAddGCPMetadataTplArgs(ctx context.Context, tc *libclient.TeleportClien
 	}
 }
 
+func maybeAddOracleOptions(ctx context.Context, tc *libclient.TeleportClient, dbInfo *databaseInfo, opts []dbcmd.ConnectCommandFunc) []dbcmd.ConnectCommandFunc {
```

Comment:
This can be limited to only dbs with oracle protocol.

Can this check be deprecated in v19?

Reply:
Yes, I'll add a `/DELETE IN` comment.

Reply:
If protocol is not Oracle, we could skip the rest of the function?

Reply:
Yes, good call!